lib/fastly_nsq/manager.rb

```diff
@@ -43,10 +43,10 @@ def stopped?
   end
 
   def add_listener(listener)
-    logger.info { "Listening to topic:'#{listener.topic}' on channel: '#{listener.channel}'" }
+    logger.info { "topic #{listener.topic}, channel #{listener.channel} listening" }
```

Comment:
`"topic #{listener.topic}, channel #{listener.channel}: listening"`

only difference is the `:`